docs/reference/api/docker_remote_api_v1.18.md

```diff
@@ -210,7 +210,7 @@ Create a container
 -   **Tty** - Boolean value, Attach standard streams to a `tty`, including `stdin` if it is not closed.
 -   **OpenStdin** - Boolean value, opens `stdin`,
 -   **StdinOnce** - Boolean value, close `stdin` after the 1 attached client disconnects.
--   **Env** - A list of environment variables in the form of `["VAR=value"[,"VAR2=value2"]]`
+-   **Env** - A list of environment variables in the form of `["VAR=value", ...]`
 -   **Labels** - Adds a map of labels to a container. To specify a map: `{"key":"value"[,"key2":"value2"]}`
```

Comment:
Hi, @thaJeztah 
I think this line has the same issue. Could you please help to modify that? 
😄